.github/pull_request_template.md

```diff
@@ -39,6 +39,8 @@ Write a short (one line) summary that describes the changes in this
 pull request for inclusion in the changelog:
 -->
 
+#### Ensure to add label/tag for the feature raised. example - [PR#2174](https://github.com/sonic-net/sonic-utilities/pull/2174) where, Generic Config and Update feature has been labelled as GCU.
```

Comment:
The example PR link actually link all future PRs to the sonic-utilities PR 2174 , and will lead to very long noise history in that PR. Please fix ASAP.

Reply:
As suggested, the example PR link has been removed and reference is given as text. And [PR#12276](https://github.com/sonic-net/sonic-buildimage/pull/12276) has been raised for the same.